crates/runtime/src/flight/flightsql/prepared_statement_query.rs

```diff
@@ -46,37 +60,30 @@ pub(crate) async fn do_action_create_prepared_statement(
 
     let schema_bytes = Service::serialize_schema(&arrow_schema)?;
 
+    let stmt = PreparedStatement {
+        query: statement.query.clone(),
+        parameters: vec![],
+    };
+
+    let handle = serde_json::to_vec(&stmt).map_err(error_to_status)?;
```

Comment:
Using JSON for this is inefficient - especially since the binary parameter data will need to be base64 encoded. Could we try with https://docs.rs/postcard as the serialization protocol? It should be more efficient than JSON.